content/en/logs/explorer/search_syntax.md

```diff
@@ -113,13 +115,43 @@ Wildcard searches work within tags and attributes (faceted or not) with this syn
 service:*mongo
 ```
 
-Wildcard searches can also be used to search in the plain text of a log that is not part of a facet. This query returns all the logs that contain the string `NETWORK`:
+Wildcard searches can also be used to search in the plain text of a log that is not part of a log attribute. For example, this query returns all logs with content (message) that contain the string `NETWORK`:
 
 ```
 *NETWORK*
 ```
 
-However, this search term does not return logs that contain the string `NETWORK` if it is in a facet and not part of the log message.
+However, this search term does not return logs that contain the string `NETWORK` if it is in a log attribute and not part of the log message.
+
+### Full-text search across all log attributes
+
+<div class="alert alert-warning">The full-text search feature is only available in Log Management and works in monitor, dashboard, and notebook queries. The full-text search syntax cannot be used for defining index filters, archive filters, log pipeline filters, or in Live Tail. </div>
```

Comment:
> works in monitor, dashboard, and notebook queries.

@siddhingra can you confirm this is correct?

Reply:
yes, this is correct